### PR TITLE
igv 2.3.93

### PR DESCRIPTION
**Pull request by ilovezfs**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by sambrightman** [comment]
Similar to #5622, the SHA does not match here. I don't see a reference to the correct SHA on their download page. The download link is also only for 2.3.92 - is it possible that formulae are being updated prematurely based on the existence of a Github release, whereas the IGV team maybe don't consider that to be authoritative (they are only based on lightweight tags)?

**Comment by ilovezfs** [comment]
@sambrightman you'll need to open an issue with upstream to find out why the checksum changed.

**Comment by sambrightman** [comment]
Why did nobody verify with them in the previous few cases? Seems like a security risk, as the checksums are not published on their homepage as far as I can tell.

See igvteam/igv#413.

**Comment by ilovezfs** [comment]
>Why did nobody verify with them in the previous few cases? 

They should have been.

**Comment by iMichka** [comment]
> Seems like a security risk

This is difficult because often we just ask upstream what is going on. And often somebody gives an explanation. But we can only trust his word. Somebody could have hijacked an abandoned github repo. 
Or when a new formula is added or a version bumped, who knows what file we are downloading ...

100% security will never be achieved. The checksum is more a check for later builds: when we commit the sha256 in our formulas, this is solely based on trust :)

**Comment by sambrightman** [comment]
Sure, someone with server access can presumably also change the published checksum. But I think always good to check what's going on. In this case, it sounds like Homebrew has been publishing pre-releases and should roll back to v2.3.92 with an updating checksum.

**Comment by wkretzsch** [comment]
Until this is fixed, does it make sense to revert 5792e81?

**Comment by wkretzsch** [comment]
Looks like we can't decrease version numbers...

**Comment by sambrightman** [comment]
You'd have to update the SHA too, I believe. Looked to me that 2.3.92 was also done too early.

**Comment by iMichka** [comment]
@wkretzsch you can, but for this you need to set `version_scheme 1` (This is like the debian epoch version numbering). This  means that the numbering is reset because of an error in the release tags. The version sheme will need to be carried forever by the formula.

**Comment by ilovezfs** [comment]
I do not think we should bump the version scheme for this.

**Comment by iMichka** [comment]
>  In this case, it sounds like Homebrew has been publishing pre-releases

We even have scripts that live-check if there is a version bump, see https://github.com/youtux/homebrew-livecheck

Homebrew has been doing this for years with all the projects. For 99% of the projects, if a new tag is pushed, we just assume this is an official release. I am not saying that the `ivg` devs are doing something wrong;  they just seem to have an uncommon workflow.

All the projects I know (that would be between 100 and 200 formulae I updated myself on a regular basis) use `beta`, `alpha` or `RC` tags when they do not want package managers to pick up their releases. Seems `ivg` is an exception to that rule.

> I do not think we should bump the version scheme for this.

I just wanted to explain this. But yes, I forgot to tell that we do not want to bump it :)

**Comment by ilovezfs** [comment]
To avoid cross-posting, I have commented here: https://github.com/igvteam/igv/issues/413#issuecomment-301830115

TL;DR: please do not retag.
